pkgs/tools/system/rex/default.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs, lib, fetchurl, perlPackages }:
+
+perlPackages.buildPerlPackage rec {
+  pname = "Rex";
+  version = "1.13.4";
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/F/FE/FERKI/Rex-1.13.4.tar.gz";
+    sha256 = "a86e9270159b41c9a8fce96f9ddc97c5caa68167ca4ed33e97908bfce17098cf";
+  };
+  buildInputs = with perlPackages; [
+    FileShareDirInstall
+    ParallelForkManager
+    StringEscape
+    TestDeep
+    TestOutput
+    TestUseAllModules
+
+    pkgs.rsync
```

Comment:
can't you add `rsync` as an input instead of `pkgs`? that's the way we normally do this.